CoreScriptsRoot/ServerCoreScripts/ServerChat/ChatServiceRunner.lua

```diff
@@ -241,6 +251,10 @@ if modules then
 	end
 end
 
+--// Wait two frames so the ChatModules have some time to initialize
+wait()
```

Comment:
additionally in this case, should specify that module initializations are not expected to wait or yield (or else stuff will break)